src/mpyl/cli/commands/health/checks.py

```diff
@@ -119,8 +120,13 @@ def __validate_config_path(
     return None
 
 
+# root_dir is necessary because we load mpyl_stages.schema.yml dynamically (but only on tests?!)
+# It would be good to move mpyl_stages back to src/mpyl/schemas (next to the other schemas) and simplify this
```

Comment:
I thought about that for a second, but then I realized that's not very simple to do as stage names are very much hardcoded in MPyL and not configurable at all:
* https://github.com/Vandebron/gh-mpyl/blob/c13c54d7b178e373003bde619fba33022877b30a/src/mpyl/steps/build/__init__.py#L4
[* https://github.com/Vandebron/gh-mpyl/blob/c13c54d7b178e373003bde619fba33022877b30a/src/mpyl/steps/test/__init__.py](https://github.com/Vandebron/gh-mpyl/blob/c13c54d7b178e373003bde619fba33022877b30a/src/mpyl/steps/test/__init__.py#L4)
* https://github.com/Vandebron/gh-mpyl/blob/c13c54d7b178e373003bde619fba33022877b30a/src/mpyl/steps/deploy/__init__.py#L4
* https://github.com/Vandebron/gh-mpyl/blob/c13c54d7b178e373003bde619fba33022877b30a/src/mpyl/steps/postdeploy/__init__.py#L4

The more I read about this "dynamic stages" feature the more I realize it was probably just started but never actually finished, I can't see how this can possibly work in the current state of MPyL.